### PR TITLE
Please use search before creating bug issue

### PR DESCRIPTION
**Pull request by turbaszek**
---
Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -24,6 +24,8 @@ Please delete these comment blocks before submitting the issue.
 
 IMPORTANT!!!
 
+PLEASE CHECK IF THIS ISSUE HAS BEEN REPORTED PREVIOUSLY USING SEARCH!!!
```
Comment (potiuk) [inline .github/ISSUE_TEMPLATE/bug_report.md:30]: I think there is a better idea. We should rather point people to the existing but not wildly know  "similar feature" functionality. When you start creating the issue and start typing description of the issue - you get "x similar issues" appear next to the "create issue" button. When you click it you will get ..... a list of similar issues... Maybe we should drag people's attention to this - it's rather natural (but I think Github Failed in making it visible enough). 


Screenshot from 2020-04-21 16-56-43


Screenshot from 2020-04-21 17-00-13

Comment (turbaszek) [reply .github/ISSUE_TEMPLATE/bug_report.md:30]: From my experience this suggestion are not state of work in terms of recommendation engine :<

Comment (turbaszek) [reply .github/ISSUE_TEMPLATE/bug_report.md:30]: From my experience this suggestion are not state of work in terms of recommendation engine :<

Comment (potiuk) [reply .github/ISSUE_TEMPLATE/bug_report.md:30]: :D. Maybe. I have rather good experiences from other projects where I tried to open issue.  I think the workflow is much more probable to happen for people. Searching moves you out from what you wanted to do (i.e. create an issue). I think from the user experience the "similar issues" are much more natural than "search first" and "create issue later". Maybe we can suggest both ?

Comment (turbaszek) [reply .github/ISSUE_TEMPLATE/bug_report.md:30]: > Maybe we can suggest both ?

 I think so

**Review by potiuk — APPROVED** [review]

**Pull request merged**